api/capi/include/nnstreamer.h

```diff
@@ -223,17 +223,17 @@ typedef enum {
  * @since_tizen 5.5
  * @remarks The @a data can be used only in the callback. To use outside, make a copy.
  * @remarks The @a info can be used only in the callback. To use outside, make a copy.
- * @param[out] data The handle of the tensor output (a single frame. tensor/tensors). Number of tensors is determined by ml_tensors_info_get_count() with the handle 'info'. Note that the maximum number of tensors is 16 (#ML_TENSOR_SIZE_LIMIT).
- * @param[out] info The handle of tensors information (cardinality, dimension, and type of given tensor/tensors).
- * @param[out] user_data User application's private data.
+ * @param[in] data The handle of the tensor input (a single frame. tensor/tensors). Number of tensors is determined by ml_tensors_info_get_count() with the handle 'info'. Note that the maximum number of tensors is 16 (#ML_TENSOR_SIZE_LIMIT).
```

Comment:
Sorry. I was a bit confused in a previous comment. This is ```... [in] ... tensor output of the pipeline```.

Reply:
Thanks. Updated.